backend/functions/consolidations/consolidations.function.test.ts

```diff
@@ -22,76 +21,81 @@ jest.mock('../lib/controllers/orders/orders.controller', () => {
 });
 
 describe('Consolidations Function tests', () => {
+  const request = createMockAzureFunctionRequest({
+    params: {
+      procedure: '',
+    },
+    method: 'PUT',
+    body: {},
+  });
+
+  const context = require('azure-function-context-mock');
```

Comment:
Ditto. Should use the az mock context helper function: createMockAzureFunctionContext.